docs/modules/servers/pages/distributed/index.adoc

```diff
@@ -12,22 +12,20 @@
 
 The Distributed Server is based on the 
 xref:servers/extendable/index.adoc[Extendable Server],
-but was built for a high-throughput distributed scenario.
-Although it does assume that you have advanced technical knowledge, 
+but was built for a large-scale distributed deployments.
+It was designed mainly for xref:concepts/???.adoc[Mail Delivery],
+but can also be used for xref:concepts/???.adoc[Mail Processing]
+Although operating a Distributed Server does assume that you have advanced 
+technical knowledge because it is the most complex and feature-rich option, 
 it is intended to be easy to operate for large deployments.
 
-This server is:
+In this section of the documentation, we will introduce you to:
 
-* Intended for use by experts only
-* Used for large-scale distributed deployments
-* Designed mainly for xref:concepts/???.adoc[Mail Delivery],
-  but can also be used for xref:concepts/???.adoc[Mail Processing]
-* The most feature-rich server, but also by far the most complex
+ * xref:distributed/context/adoc[The context of a large-sacle distributed email system]
+ * xref:distributed/objectives.adoc[Objectives and motivation of the Distributed Server]
+ * xref:distributed/concepts.adoc[Useful background concepts]
+ * xref:distributed/architecture.adoc[Overview of architecture]
```

Comment:
As stated above, I do think architecture means "concepts" adapted to the Distributed server. I don't see the need of splitting them.

Reply:
I see.

The idea was to actually describe some of the underlying concepts in more detail, maybe with a paragraph or two, and links to relevant external documents. The "concepts" are mostly terms that the James community did not invent, but are important concepts used to architect, design, and implement James. My thought is that we should not assume that everybody knows all concepts well, so they would appreciate having a small introduction.

The architecture is 100% James. It shows the thinking behind how James was designed. If possible, this page should be very visual.